### DISCUSSION TITLE
Change sea_floor interaction

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#1028] [Q&A]
Hi,

 I am studying a channel V-shaped, with steep bathymetry and I used a random vertical distribution of z to seed particles. After that

o.set_config('general:seafloor_action', 'deactivate')

was used to "remove" those seeded below the seafloor, however this option also deactivates particles that had a "good" 
initial position, if they were advected into the sea floor.

Any advice on how to seed to a section with an arbitrary vertical shape? I would like to use

o.set_config('general:seafloor_action', 'previous')

**Comment by gauteh** [comment]
I haven't tried this myself, but maybe you can manually call: interact_with_seafloor (`https://github.com/OpenDrift/opendrift/blob/a1c747d6eacac2b299360a8574a129da7bdb2d10/opendrift/models/basemodel.py#L815`) after setting the config, and then reset it before starting the simulation.
